packages/scss/src/components/multiSelect/mods.scss

```diff
@@ -4,40 +4,41 @@
 @use '@lucca-front/scss/src/components/numericBadge/exports' as numericBadge;
 
 @mixin S {
-  --components-multiSelect-fontSize: var(--sizes-S-fontSize);
-  --components-multiSelect-lineHeight: var(--sizes-S-lineHeight);
-  --components-multiSelect-padding: var(--pr-t-spacings-75);
-  --components-multiSelect-gap: var(--pr-t-spacings-75);
-
-  .multipleSelect-arrow {
-    @include icon.S;
-  }
-
-  .multipleSelect-clear {
-    @include clear.S;
-  }
-
-  .multipleSelect-displayer-chip {
-    @include chip.S;
-  }
-
-  .multipleSelect-displayer-numericBadge {
-    @include numericBadge.S;
-  }
+	--components-multiSelect-fontSize: var(--sizes-S-fontSize);
+	--components-multiSelect-lineHeight: var(--sizes-S-lineHeight);
+	--components-multiSelect-padding: var(--pr-t-spacings-75);
+	--components-multiSelect-gap: var(--pr-t-spacings-75);
+
+	.multipleSelect-arrow {
+		@include icon.S;
+	}
+
+	.multipleSelect-clear {
+		@include clear.S;
+	}
+
+	.multipleSelect-displayer-chip {
+		@include chip.S;
+	}
+
+	.multipleSelect-displayer-numericBadge {
+		@include numericBadge.S;
+	}
 }
 
 @mixin displayerFilter {
-  display: grid;
-
-  .multipleSelect-displayer-filter {
-    grid-column-start: 1;
-    grid-row-start: 1;
-    display: flex;
-    gap: var(--pr-t-spacings-50);
-  }
-
-  .multipleSelect-displayer-search {
-    grid-column-start: 1;
-    grid-row-start: 1;
-  }
+	display: grid;
+
+	.multipleSelect-displayer-filter {
+		grid-column-start: 1;
+		grid-row-start: 1;
+		display: flex;
+		gap: var(--pr-t-spacings-50);
+		min-width: 0;
+	}
+
+	.multipleSelect-displayer-search {
+		grid-column-start: 1;
+		grid-row-start: 1;
+	}
```

Comment:
Are we supposed to use tabs?

Reply:
It's not yet clear on the LF side.

And if I have an opinion, it's here: https://adamtuttle.codes/blog/2021/tabs-vs-spaces-its-an-accessibility-issue/

Reply:
I don’t personally remember why I switched to spaces over tabs, so I won’t mind either, and the accessibility argument is a strong one. It would be good if we could code with the same logic though, or things might get complicated. :)

Shall this be part of the guidelines?

![image](https://github.com/LuccaSA/lucca-front/assets/144820795/7ef14b6a-6ddc-4899-8933-25663256cbf9)

(source: https://www.emacswiki.org/emacs/TabsSpacesBoth)